frontend/test/src/reducer/user.spec.js

```diff
@@ -0,0 +1,55 @@
+import { expect } from 'chai'
+import userReducer from '../../../src/reducer/user.js'
+import {
+  UPDATE,
+  USER_USERNAME,
+  USER_PUBLIC_NAME
+} from '../../../src/action-creator.sync'
+import { getBrowserLang } from '../../../src/helper.js'
+import { PROFILE } from 'tracim_frontend_lib'
+
+const defaultUser = {
```

Comment:
your test user should have actual values.
There is a user in the fixture folder, you can use it.

Check that this user has all the properties and update it if required

Reply:
The purpose of defaultUser was to test that the reducer "should return the default state", it is equal to the default state, so it has no values.
For the other tests I will use the user in the fixture folder, but for this "it" test I still think it is important to keep the defaultUser.

Reply:
I see. Then you need to get the defaultUser variable from the reducer's file

Reply:
export it if it is required